### DISCUSSION TITLE
REMOTE_AUTH_AUTO_CREATE_USER is being ignored.

### DISCUSSION THREAD

**Discussion by appleoddity** [netbox-community/netbox-docker#1053] [Getting Help]
I'm implementing Azure AD SSO as described here: https://docs.netbox.dev/en/stable/administration/authentication/microsoft-azure-ad/

The environment variable `REMOTE_AUTH_AUTO_CREATE_USER` is being ignored. Even though I set it to false, a new user is created whenever I perform an SSO login through Azure AD.

I tried setting `REMOTE_AUTH_AUTO_CREATE_USER=False` in `env/netbox.env` and in a `configuration/extra-settings.py` file.  I've even tried changing the default setting in `configuration.py` (mentioned below).

In addition, the documentation seems to be different from reality. All documentation I can find suggests this setting defaults to false. But, upon closer inspection of `configuration.py` (line 277) it is defaulting to `True`:
`REMOTE_AUTH_AUTO_CREATE_USER = _environ_get_and_map('REMOTE_AUTH_AUTO_CREATE_USER', 'True', _AS_BOOL)`

**Comment by shiftycow** [comment]
Confirmed that this is still a bug. Setting `REMOTE_AUTH_AUTO_CREATE_USER = False` in configuration.py is ignored, the flag always gets set to True somewhere.
